operators/arithmetic.js

```diff
@@ -4,21 +4,21 @@ let number1 = 5;
 let number2 = 3;
 
 // Addition
-console.log('5 + 3 = ' , number1 + number2);
+console.log(`${number1} + ${number1} = ${number1 + number2}`);
 
 // Subtraction
-console.log('5 - 3 = ' , number1 - number2);
+console.log(`${number1} - ${number1} = ${number1 - number2}`);
 
 // Multiplication
-console.log('5 * 3 = ' , number1 * number2);
+console.log(`${number1} * ${number1} = ${number1 * number2}`);
 
 // Division
-console.log('5 / 3 = ' , number1 / number2);
+console.log(`${number1} / ${number1} = ${number1 / number2}`);
 
 // Remainder
-console.log('5 % 3 = ' , number1 % number2);
+console.log(`${number1} % ${number1} = ${number1 % number2}`);
 
 // Exponentiation
-console.log('5 ** 3 = ', number1 ** number2);
+console.log(`${number1} ** ${number1} = ${number1 ** number2}`);
 
```

Comment:
From :
console.log(`${number1} + **${number1}** = ${number1 + number2}`);
To :
console.log(`${number1} + **${number2}** = ${number1 + number2}`);